ember-aria-utilities/src/test-support/-private/aria-grid.ts

```diff
@@ -152,6 +152,6 @@ const keys = {
 
 export const ariaGrid = {
   keys,
-  gridSelectors,
```

Comment:
if you do intend to move to `gridSelectors`, you could do:

```js
export const ariaGrid = {
  keys,
  selectors,
  gridSelectors,
  tableSelectors,
}
```

and if we want to deprecate `selectors`, you could do:

```js
let notified = false;
export const ariaGrid = {
  keys,
  get selectors() {
    if (!notified) {
      deprecate('message about using gridSelectors insetad', ...);
      notified = true;
    }
    return selectors;
  }
  gridSelectors,
  tableSelectors,
}
```